plugins/module_utils/network/eos/config/lacp_interfaces/lacp_interfaces.py

```diff
@@ -240,9 +240,14 @@ def _state_deleted(want, have):
 def generate_commands(interface, to_set, to_remove):
     commands = []
     for key in to_remove.keys():
+        if key == "rate":
+            key = "timer"
```

Comment:
Do you still need these with the alias swapped around? I don't think anything should be setting `key` to "rate"